tool/tas.re

```diff
@@ -209,4 +209,4 @@ static TAS_COMPONENT = Component {
         TAS_STATE.events.push(InputEvent::MouseMoved(x, y));
     },
     on_component_exit: fn() {},
-};
+};
```

Comment:
Empty lines at the end of files are convention. Even if they weren't, or you were adding a missing newline here, if that's the only change to the whole file, don't do it in this PR.